cmd/util/cmd/find-inconsistent-result/cmd.go

```diff
@@ -179,15 +180,20 @@ func findLastExecutedAndSealedHeight(state protocol.State, db *badger.DB) (uint6
 	}
 
 	var blockID flow.Identifier
-	var lastExecuted uint64
-	err = db.View(procedure.GetLastExecutedBlock(&lastExecuted, &blockID))
+	reader := badgerimpl.ToDB(db).Reader()
+	err = operation.RetrieveExecutedBlock(reader, &blockID)
```

Comment:
`findLastExecutedAndSealedHeight()` (not added in this PR) appears to be very similar to  `latest.LatestSealedAndExecutedHeight()`.  Maybe reuse that function.

https://github.com/onflow/flow-go/blob/516365e9f5f1bee3e14c5507217f0a8585f11eca/module/block_iterator/latest/sealed_and_executed.go#L63